### PR TITLE
add more sqs tests to prepare for new sqs implementation

### PR DESCRIPTION
**Pull request by thrau**

### CODE REVIEW

File: tests/integration/test_sqs.py [hunk 1]
```diff
@@ -1103,3 +1104,277 @@ def check_msg_attributes(self, queue_url, attrs):
...
+
+        # list queues with first prefix
+        result = sqs_client.list_queues(QueueNamePrefix="a-test-queue-")
+        assert "QueueUrls" in result
+        assert len(result["QueueUrls"]) == 2
+        assert queue_urls[0] in result["QueueUrls"]
+        assert queue_urls[1] in result["QueueUrls"]
+        assert queue_urls[2] not in result["QueueUrls"]
+
+        # list queues with first prefix
```
Comment (dfangl) [inline tests/integration/test_sqs.py:1133]: With the second prefix?

File: tests/integration/test_sqs.py [hunk 2]
```diff
@@ -1103,3 +1104,277 @@ def check_msg_attributes(self, queue_url, attrs):
...
+            Entries=[
+                {"Id": "1", "MessageBody": "message-0"},
+                {"Id": "2", "MessageBody": "message-1"},
+            ],
+        )
+        sqs_client.send_message(QueueUrl=sqs_queue, MessageBody="message-2")
+
+        response = sqs_client.receive_message(QueueUrl=sqs_queue, MaxNumberOfMessages=3)
+        assert len(response["Messages"]) == 3
+        print(response["Messages"][0])
```
Comment (dfangl) [inline tests/integration/test_sqs.py:1221]: same as above, any reason for the prints?

File: tests/integration/test_sqs.py [hunk 3]
```diff
@@ -1103,3 +1104,277 @@ def check_msg_attributes(self, queue_url, attrs):
...
+        sqs_client.send_message(QueueUrl=queue0, MessageBody="message")
+
+        result = sqs_client.receive_message(QueueUrl=queue1)
+        assert "Messages" not in result
+
+        result = sqs_client.receive_message(QueueUrl=queue0)
+        assert len(result["Messages"]) == 1
+        assert result["Messages"][0]["Body"] == "message"
+
+        print(result["Messages"])
```
Comment (dfangl) [inline tests/integration/test_sqs.py:1183]: any reason for this print here?

**Review by dfangl — APPROVED** [review]
Nice :+1:

File: tests/integration/test_sqs.py (hunk 1 above)
Comment (thrau) [reply tests/integration/test_sqs.py]: good catch!

**Pull request merged**